src/bindings/python/src/pyopenvino/graph/layout_helpers.cpp

```diff
@@ -23,4 +23,6 @@ void regmodule_graph_layout_helpers(py::module m) {
     mod.def("height_idx", &ov::layout::height_idx, py::arg("layout"));
     mod.def("has_width", &ov::layout::has_width, py::arg("layout"));
     mod.def("width_idx", &ov::layout::width_idx, py::arg("layout"));
+    mod.def("get_layout", (ov::Layout(*)(const ov::Output<const ov::Node>&)) & ov::layout::get_layout, py::arg("port"));
+    mod.def("set_layout", &ov::layout::set_layout, py::arg("port"), py::arg("layout"));
```

Comment:
please, add python tests as well :)

Reply:
@AlexeyLebedev1 Could you provide the right place for python tests?

Reply:
Tests were added